libretro/libretro-core.c

```diff
@@ -1890,6 +1853,8 @@ static void update_variables(void)
         cbm2model_set(modl);
    }
 #else
+   bool resid_model_changed = false;

```

Comment:
You are declaring this only when vic20 is not defined but use it later without the ifdef guard.

Reply:
VIC 20 should have no SID anyway, there was no guard in the original code for SID-systems but I think there should be one like in ui.c:266 as seen in the patch below. Shouldn't that guard also be there for the SID core-options? I'll take care of the guards - can I then replace the commit in the PR or needs a new one to be created?

Reply:
Thanks you can just make the changes then git commit --amend them and then force push to your branch. This Pr will automatically update then